### PR TITLE
WT-5212 Incremental backup data validation test

### PR DESCRIPTION
**Pull request by raviprakashgiri29**

### CODE REVIEW

File: test/suite/test_backup13.py [hunk 1]
```diff
@@ -0,0 +1,368 @@
...
+    def compare_backups(self, t_uri):
+        #
+        # Run wt dump on full backup directory
+        #
+        full_backup_out = self.full_out + '.' + str(self.counter)
+        if self.counter == 0:
+            self.runWt(['-R', '-h', self.home, 'dump', t_uri], outfilename=full_backup_out)
+        else:
+            home_dir = self.home_full + '.' + str(self.counter)
+            self.runWt(['-R', '-h', home_dir, 'dump', t_uri], outfilename=full_backup_out)
```
Comment (sueloverso) [inline test/suite/test_backup13.py]: Similar to the above, could we just have the conditional set the variable and simplify this.

**Comment by sueloverso** [comment]
@raviprakashgiri29 please name your test file `test_backup14.py` as I have a soon-to-be merged branch that adds a `test_backup13.py` for WT-5214.

**Comment by sueloverso** [comment]
Please review what failed in PR testing. If it is DH related and not related to this, I'm okay if you merge (once the test rename and minor things are addressed).

**Comment by raviprakashgiri29** [comment]
> Please review what failed in PR testing. If it is DH related and not related to this, I'm okay if you merge (once the test rename and minor things are addressed).

Thank you @sueloverso for reviewing the changes. The PR testing failed with some other reason and it is passed now. I will address the review comments and merge the changes.

File: test/suite/test_backup13.py [hunk 2]
```diff
@@ -0,0 +1,368 @@
...
+import os, shutil
+from helper import compare_files
+from suite_subprocess import suite_subprocess
+from wtdataset import simple_key
+from wtscenario import make_scenarios
+import glob
+
+# test_backup13.py
+# Test cursor backup with a block-based incremental cursor.
+class test_backup14(wttest.WiredTigerTestCase, suite_subprocess):
```
[the remark this answers is not in the document]
Comment (raviprakashgiri29) [reply test/suite/test_backup13.py]: Fixed

File: test/suite/test_backup13.py [hunk 3]
```diff
@@ -0,0 +1,368 @@
...
+
+        #
+        # First time through we take a full backup into the incremental directories. Otherwise only
+        # into the appropriate full directory.
+        #
+        if self.initial_backup == True:
+            buf = 'incremental=(granularity=1M,enabled=true,this_id=ID0)'
+            cursor = self.session.open_cursor('backup:', None, buf)
+        else:
+            cursor = self.session.open_cursor('backup:', None, None)
```
[the remark this answers is not in the document]
Comment (raviprakashgiri29) [reply test/suite/test_backup13.py]: Fixed

File: test/suite/test_backup13.py (hunk 1 above)
Comment (raviprakashgiri29) [reply test/suite/test_backup13.py]: Fixed

**Pull request merged**